src/pyload/plugins/containers/DLC.py

```diff
@@ -5,59 +5,42 @@
 import re
 import xml.dom.minidom
 
-from cryptography.hazmat.backends import default_backend
-from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+import Crypto.Cipher.AES
```

Comment:
Are you sure to go back to `Crypto` instead of `cryptography`?

Reply:
Changed, to right version